### DISCUSSION TITLE
New WiFi - Change SSID/Password while connected to NSPanel as AP

### DISCUSSION THREAD

**Discussion by belgracing** [Blackymas/NSPanel_HA_Blueprint#1121] [General]
I moved my HA to a new location. The WiFi SSID and password are different there. Is there a way to edit the WiFi settings of the NSPanel? I could connect to it while it is working as AP but what should I do next?
Thanks in advance!

**Comment by edwardtfn** [comment]
You have to change your Wi-Fi settings in substitutions in your yaml, but as that is also used for OTA password, you have to do something like this:

https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/en/customization.md#custom-ota-password

**Reply by belgracing** [reply]
Thanks. I just wanted to edit the settings via WiFi if possible. I supposed the WiFi settings are accessible when you connect to the NSPanel working as the AP.

However, it seems the only option is to disassemble the device and flash it using the usb to ttl serial adapter again :(

**Reply by edwardtfn** [reply]
No, if you still have access to the device you can still do this.
You can build the firmware file using the instructions I gave before and when installing chose to download to your computer, than you can connect to WiFi and upload that.
No need to disassemble.
